test_app/lib/screens/join_screen.dart

```diff
@@ -153,62 +154,88 @@ class _JoinConferenceContentState extends State<JoinConferenceContent> {
                       controller: conferenceAliasTextController,
                       focusColor: Colors.deepPurple),
                 ),
-                ExpansionTile(title: const Text('Options'),
+                ExpansionTile(
+                    title: const Text('Options'),
                     textColor: Colors.black,
                     iconColor: Colors.deepPurple,
                     children: [
-                  SwitchOption(
-                      title: 'Observe Conference Status',
-                      value: switchConferenceStatus,
-                      onChanged: (value) {
-                        setState(() {
-                          switchConferenceStatus = value;
-                          observeConferenceStatus(switchConferenceStatus);
-                        });
-                      }),
-                  SwitchOption(
-                      title: 'Dolby Voice',
-                      value: switchDolbyVoice,
-                      onChanged: (value) {
-                        if (value == false) {
+                      SwitchOption(
+                          title: 'Observe Conference Status',
+                          value: switchConferenceStatus,
+                          onChanged: (value) {
+                            setState(() {
+                              switchConferenceStatus = value;
+                              observeConferenceStatus(switchConferenceStatus);
+                            });
+                          }),
+                      SwitchOption(
+                          title: 'Dolby Voice',
+                          value: switchDolbyVoice,
+                          onChanged: (value) {
+                            if (value == false) {
+                              setState(() {
+                                switchDolbyVoice = value;
+                                spatialAudio = value;
+                              });
+                            } else {
+                              setState(() {
+                                switchDolbyVoice = value;
+                              });
+                            }
+                          }),
+                      DropdownButton<String>(
+                        focusColor: Colors.white,
+                        value: switchDolbyVoice? spatialAudioStyleText : null,
+                        style: const TextStyle(color: Colors.white),
+                        iconEnabledColor: Colors.black,
+                        items: <String>[
+                          'Spatial Audio with Individual',
+                          'Spatial Audio with Shared Scene',
```

Comment:
I thought the spatial audio style enum had 3 values.

Reply:
Oh yes, I missed adding the last one. I will add it. thank you.